code/modules/mafia/controller.dm

```diff
@@ -1,35 +1,62 @@
+
+
+/**
+  * The mafia controller handles the mafia minigame in progress.
+  * It is first created when the first ghost signs up to play.
+  */
 /datum/mafia_controller
+	///all roles in the game, dead or alive. check their game status if you only want living or dead.
 	var/list/all_roles = list()
-	var/list/player_role_lookup = list() //This only exists to speed up role retrieval
+	///exists to speed up role retrieval, it's a dict. player_role_lookup[player ckey] will give you the role they play
+	var/list/player_role_lookup = list()
+	///what part of the game you're playing in. day phases, night phases, judgement phases, etc.
 	var/phase = MAFIA_PHASE_SETUP
+	///how long the game has gone on for, changes with every sunrise. day one, night one, day two, etc.
 	var/turn = 0
 
-	var/first_day_phase_period = 20 SECONDS //say hi to everyone, no voting so no discussion
-	var/day_phase_period = 1 MINUTES //talk with others about the night
-	var/voting_phase_period = 30 SECONDS //vote someone to get put on trial
-	var/judgement_phase_period = 30 SECONDS //defend yourself! don't get lynched! sometimes skipped
-	var/judgement_lynch_period = 5 SECONDS //just watch a guy die (or live but lol)
-	var/night_phase_period = 45 SECONDS //mafia talk and pick someone to kill
-	var/victory_lap_period = 20 SECONDS //just waiting around sayin' woop woop
-
+	///first day has no voting, and thus is shorter
+	var/first_day_phase_period = 20 SECONDS
+	///talk with others about the last night
+	var/day_phase_period = 1 MINUTES
+	///vote someone to get put on trial
+	var/voting_phase_period = 30 SECONDS
+	///defend yourself! don't get lynched! sometimes skipped if nobody votes.
+	var/judgement_phase_period = 30 SECONDS
+	///guilty or innocent, we want a bit of time for players to process the outcome of the vote
+	var/judgement_lynch_period = 5 SECONDS
+	///mafia talk at night and pick someone to kill, some town roles use their actions, etc etc.
+	var/night_phase_period = 45 SECONDS
+	///like the lynch period, players need to see what the other players in the game's roles were
+	var/victory_lap_period = 20 SECONDS
+
+	///template picked when the game starts. used for the name and desc reading
 	var/datum/map_template/mafia/current_map
+	///map generation tool that deletes the current map after the game finishes
 	var/datum/map_generator/massdelete/map_deleter
 
-	var/list/current_setup_text //Redable list of roles in current game
+	///Readable list of roles in current game (Psychologist x1, Clown x2, etc)
```

Comment:
```suggestion
	///Readable list of roles in current game (Psychologist = 1, Clown = 2, etc)
```
??

Reply:
This is literally what is being sent, a list of "Psychologist x1", i'll put quotes to make it clearer